_assets/stylesheets/pages/_community-store.scss

```diff
@@ -0,0 +1,150 @@
+#community-store-sub-nav {
+
+    width: 100%;
+    height: 44px;
+    background-color: $cr-cyan;
+    color: #ffffff;
+    display: flex;
+
+    .container {
+      padding-left: 30px;
+
+      span  {
+        padding-right: 24px;
+        line-height: 44px;
+        vertical-align: middle;
+        flex: 0 0 auto; 
+
+        a {
+            font-size: 14px;
+            color: #ffffff;
+        }
+      }
+    }
+}
+
+#community-store-intro {
+
+    height: 597px;
+    display: flex;
+    padding-bottom: 87px;
+    margin-bottom: 0;
+
+    .container {
+
+        align-self: flex-end;
+
+        .row {
+        text-align: left;
+        max-width: 100%;
+
+        .title {
+            font-family: $condensed-extra-font-face;
+            font-weight: 600;
+            font-size: 5.5rem;
+            line-height: 5rem;
+            color: #ffffff;
+            letter-spacing: 2px;
+            margin-bottom: 20px;
+        }
+    
+        p {
+            opacity: 0.8;
+            font-size: 19px;
+            font-weight: 300;
+            line-height: 1.42;
+            color: #ffffff;
+            margin-bottom: 0px;
+        }
+    }
+    }
+}
+
+.card-container {
+    a {
+        color: $cr-cyan;
+    }
+
+    @media (min-width: 993px){
```

Comment:
Please use the variables provided by Bootstrap for the calculation stuff. `$screen-md` for example is `992px` so you could probably do this... 

```
@media (min-width: $screen-md + 1){ ... }
```